classes/Provider/Gateways/WebGatewayProvider.php

```diff
@@ -49,6 +51,20 @@
 {
     public function register(Container $app)
     {
+        $app['mailer'] = function ($app) {
+            $options = [
```

Comment:
What do we need `$options` for if we initialize it, and then access its values?

Can we skip the step and just access the values directly?

Can we just get rid of